engine/render/renderer/renderer/BillboardRenderer.cpp

```diff
@@ -44,12 +36,12 @@ void BillboardRenderer::Initialise(const String& globalDataAttributeName,
 
 void BillboardRenderer::RecreateMaterials()
 {
-    billboardMaterial.RecreatePipeline();
+    billboardMaterial.Recreate();
 }
 
 void BillboardRenderer::Destroy()
 {
-    billboardMaterial.DestroyMaterial();
+    billboardMaterial.~Material();
```

Comment:
`destroy billboardMaterial`?

Reply:
I have no idea what possessed me to do that. I've remade the `Destroy` function for this.